app/pathogen/arbovirus/dashboard/filters.tsx

```diff
@@ -171,11 +172,20 @@ interface FiltersProps {
 }
 
 export function Filters(props: FiltersProps) {
+  const { selectedFilters } = useContext(ArboContext);
+
   const excludedFields = props.excludedFields ?? [];
 
+  const selectedAgeGroups = selectedFilters['ageGroup'] ?? [];
+
+  if(!(selectedAgeGroups.length === 1 && selectedAgeGroups[0] === 'Children and Youth (0-17 years)')) {
+    excludedFields.push(FilterableField.pediatricAgeGroup);
+  }
```

Comment:
100% agree not ideal to be hardcoding this stuff in the front end because if it changes in Airtable it will break the pediatric age group filter. I definitely don't think this is the last iteration of this, I just want to give us something to iterate on lol

Reply:
An interesting consequence of the way our filter system works is that I can't say "filter on age group (0-4) and age group (18-64)" because that would involve adding the pediatric age group filter and it would filter out all the studies with a pediatric age group of null which would be all of the (18-64) ones.

Thought that was kind of interesting. I'm not implementing a fix in this PR because I think this PR is big enough as-is but thought I would mention it for an interesting feature we might want to consider later.

Reply:
I bet #106 would need to basically work the same way as this except with DENV/pathogen instead of 0-17/Age group so that's maybe an interesting time to copy-paste this code and maybe see if we can make something more generalized once there's two copies of the code? Just kind of thinking out loud lol

Reply:
When we're doing SarsCoV2 stuff, I'm going to see if there's maybe some kind of thing we can do in the backend for things like this? I'd have to give it some more thought hahaha

Reply:
All sounds good to me, and yeah for now it is a-ok. I like the idea of coming back to it when we do the DENV 1 2 3 4 stuff